quint/src/cli.ts

```diff
@@ -238,7 +238,12 @@ const verifyCmd = {
       .option('apalache-config', {
         desc: 'Filename of the additional Apalache configuration (in the HOCON format, a superset of JSON)',
         type: 'string',
-      }),
+      })
+      .option('verbosity', {
+        desc: 'control how much output is produced (0 to 5)',
+        type: 'number',
+      })
+      .default('verbosity', verbosity.defaultLevel),
```

Comment:
The default can be supplied in the option dict, as on lines 221 and 226 above:

```suggestion
      .option('verbosity', {
        desc: 'control how much output is produced (0 to 5)',
        type: 'number',
        defualt: verbosity.defaultLevel
      })
```